src/Symfony/Component/Security/Http/Firewall/LegacyListenerTrait.php

```diff
@@ -24,15 +23,15 @@ trait LegacyListenerTrait
     /**
      * @deprecated since Symfony 4.3, use __invoke() instead
```

Comment:
hello deprecation (the class itself is deprecated)

Reply:
going to create a PR for this.